packages/financial-templates-lib/test/clients/OptimisticOracleClient.js

```diff
@@ -6,17 +6,15 @@ const { OptimisticOracleClient } = require("../../src/clients/OptimisticOracleCl
 const { interfaceName, advanceBlockAndSetTime } = require("@uma/common");
 const { getTruffleContract } = require("@uma/core");
 
-const CONTRACT_VERSION = "latest";
```

Comment:
this one might be a bit controversial. IMHO it's redundant to define this. it's a fair bit cleaner to remove it wholesale. what do you think @UMAprotocol/eng ?

Reply:
@nicholaspai may have stronger opinions, but I'm totally fine relying on the default arg.

Reply:
I am indifferent. In the future if we want to move off of latest we can easily change this variable, so I'd lean slightly towards keeping it.

Reply:
for now, lets stay on the side of defaulting unless tests explicitly execute on a specific version.